processor/src/test/java/org/mapstruct/ap/test/nestedbeans/exceptions/_target/DeveloperDto.java

```diff
@@ -0,0 +1,86 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.test.nestedbeans.exceptions._target;
+
+/**
+ * @author Filip Hrisafov
+ * @author Darren Rambaud
+ */
+public class DeveloperDto {
```

Comment:
Are all these properties needed to demonstrate the problem (both in parent and child)? They confused me initially..

Reply:
You are right. Not everything is needed. I'll remove all except one

Reply:
Great. I think we can save the example for other cases (I'm always out of inspiration when I need test cases and hence end up with complete aquaria).

Reply:
Actually @xyzst made a more extensive example [here](https://github.com/xyzst/unwrapped-exceptions). Maybe he can contribute it to our examples repo if he is up for it :)